common/src/main/java/com/wynntils/core/crowdsource/type/CrowdSourcedDataType.java

```diff
@@ -6,13 +6,19 @@
 
 import com.google.common.base.CaseFormat;
 import com.wynntils.core.crowdsource.datatype.LootrunTaskLocation;
+import com.wynntils.models.npc.label.NpcLabelInfo;
+import com.wynntils.models.profession.label.GatheringNodeLabelInfo;
+import com.wynntils.models.profession.label.GatheringStationLabelInfo;
 import net.minecraft.client.resources.language.I18n;
 
 /**
  * This enum represents the type of crowd sourced data that is being collected.
  */
 public enum CrowdSourcedDataType {
-    LOOTRUN_TASK_LOCATIONS(LootrunTaskLocation.class);
+    LOOTRUN_TASK_LOCATIONS(LootrunTaskLocation.class),
+    NPC_LOCATIONS(NpcLabelInfo.class),
+    PROFESSION_NODE_LOCATIONS(GatheringNodeLabelInfo.class),
+    PROFESSION_STATION_LOCATIONS(GatheringStationLabelInfo.class);
```

Comment:
Not sure I'm a fan of these names. I think they should be Crafting Stations

Reply:
I want to have the profession prefix. Profession Crafting Station should be a good choice.